### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by luraess**
- Provide a data download function
- Refactor `fetch_glacier`
- Change output names (glacier name first)
- Add example to read in a Dict

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,14 @@ SGI_IDS = Dict( "Rhone"       => "B43-03",
...
                 "ArollaHaut"  => "B73-12")
 ```
+
+An alternative workflow is to first fetch the data for the entire Alps (the download step) and then select the geometry of a list of glacier given in the `SGI_IDS` dict automatically as following:
+```
+datadir = joinpath(@__DIR__,"mydata/")
+fetch_data(datadir)
+data = geom_select.(keys(SGI_IDS), values(SGI_IDS), datadir);
+```
+
```
Comment (mauro3) [inline README.md:58]: Is there a way to get a dict of all SGI-glaciers?  If so, would be good to document it here.

**Comment by mauro3** [comment]
Is this a breaking change? If so, best discuss it briefly with Christophe.

**Comment by luraess** [comment]
No breaking change AFAICT

File: README.md (hunk 1 above)
Comment (luraess) [reply README.md:58]: Don't know. Maybe @christopheogier found something?

**Comment by mauro3** [comment]
Probably should merge this?

**Comment by luraess** [comment]
Yes

**Comment by luraess** [comment]
This PR should additionally close #13 upon merge.

**Pull request merged**